prow/cluster/resources/monitoring/dashboards/grafana-prow.json

```diff
@@ -808,7 +805,6 @@
         }
       ],
       "thresholds": "200,500",
-      "timeFrom": "12h",
```

Comment:
I would left this last "timeFrom" as it's for background chart for Github token quota usage. Without this chart show data for last hour which i useless as there is no scale and value displayed for the chart.
<img width="695" alt="Screenshot 2020-06-04 at 15 05 50" src="https://user-images.githubusercontent.com/12400578/83760537-612dd700-a675-11ea-836e-c988e8c8d387.png">

Reply:
That's a functionality of this panel. The spark lines should show the value change over the given time period. We can't easily define min and max for background lines. The background will scale relatively for given time period. That means if 3000 requests is the lowest value for given time period it will be displayed as the lowest point on the background chart. Here's an issue for that https://github.com/grafana/grafana/issues/11949

Maybe creating another graph for token usage history where we could add series would be ok?

Reply:
If we target on getting historical data for this metric we should create another chart. I've added this as additional representation of data for longer time and to make it nice.  It's not needed for the panel to function. Go ahead and change it to another chart, if you would like. You can leave it without "timeFrom" if the timeframe for spark lines will change when you change the timeframe for the dashboard.

Reply:
Done. The Y-axis is fixed to range between 5000-0. It should be more readable for any of the time range.
<img width="961" alt="image" src="https://user-images.githubusercontent.com/23667947/84008044-89694e80-a971-11ea-89b8-4f9c775b0180.png">